src/ontology/mondo-ingest.Makefile

```diff
@@ -246,6 +257,7 @@ $(REPORTDIR)/%_term_exclusions.txt $(REPORTDIR)/%_exclusion_reasons.robot.templa
 	--config-path metadata/$*.yml \
 	--outpath-txt $(REPORTDIR)/$*_term_exclusions.txt \
 	--outpath-robot-template-tsv $(REPORTDIR)/$*_exclusion_reasons.robot.template.tsv
+.PRECIOUS: $(REPORTDIR)/%_exclusion_reasons.robot.template.tsv
```

Comment:
## Bug fix: `rm reports/icd11foundation_exclusion_reasons.robot.template.tsv`
- [ ] Review and see questions @matentzn @twhetzel

AFAIK, as of 2024/03/20, this is the only remaining issue that we may want to resolve before merging. That's the only reason why this is still a draft (though #471 is not).

As stated in [the thread about the slurp bug fix](https://github.com/monarch-initiative/mondo-ingest/pull/434#discussion_r1533102897), I noticed at the end that `build-mondo-ingest` that this file was also being removed.

## Questions
Point (1) and (2) in the thread linked above also apply here.

But here is another question that only applies to this case:
If there are two targets in the goal `$(REPORTDIR)/%_term_exclusions.txt $(REPORTDIR)/%_exclusion_reasons.robot.template.tsv`, why is only 1 of them getting removed?

Reply:
I have no idea, but this is a good question - for you to answer by yourself. Not for this PR though :P

Reply:
Ok. I talked over it with Trish and she doesn't know either. ChatGPT also doesn't know. This will remain a mystery.

I'm going to continue adding tangent discussions to my PRs, because these are also supposed to get reviewed at meetings and this is the most time efficient and easiest, most contextual way for me to go through these discussions. If I add them to Workflowy or Google Docs, they just get buried and I can't remember the context months later.